### PR TITLE
drm: improve wl_output.description

### PR DESCRIPTION
**Pull request by emersion**
Use a more human-readable string.

### CODE REVIEW

**Comment by Joshua-Ashton** [comment]
We already have something parsing edid there, better to just use that?

File: src/drm.hpp [hunk 1]
```diff
@@ -68,6 +68,9 @@ struct connector {
 	uint32_t possible_crtcs;
 	std::map<std::string, const drmModePropertyRes *> props;
 	std::map<std::string, uint64_t> initial_prop_values;
+
+	char *make;
+	char *model;
```
Comment (slouken) [inline src/drm.hpp:73]: Where do these get initialized?

**Comment by emersion** [comment]
> We already have something parsing edid there, better to just use that?

Ah, missed it. Done.

File: src/drm.hpp (hunk 1 above)
Comment (emersion) [reply src/drm.hpp:74]: Zero-initialized in `refresh_state`, with

```c
struct connector conn = { .id = conn_id };
```

Populated in `parse_edid` when the connector status switches to "connected".

**Pull request merged**